### PR TITLE
Add note about fuzzy and fix heading nesting

### PR DESCRIPTION
**Pull request by pierd**

### CODE REVIEW

**Comment by pierd** [comment]
Just submitted the CLA.

**Review by mgeisler — APPROVED** [review]
Excellent, thank you!

**Comment by mgeisler** [comment]
> Just submitted the CLA.

Thanks, looks good!

File: TRANSLATIONS.md [hunk 1]
```diff
@@ -102,7 +107,7 @@ This will update the book's language to `xx`, it will make the `mdbook-gettext`
 preprocessor become active and tell it to use the `po/xx.po` file, and finally
 it will redirect the output to `book/xx`.
 
-## Serving a Translation
+### Serving a Translation
```
Comment (mgeisler) [inline TRANSLATIONS.md:110]: Oh, I hadn't noticed. Thanks for fixing!

**Pull request merged**